index.js

```diff
@@ -38,8 +38,8 @@ module.exports = function(content) {
 	var outputPath = "";
 
 	var filePath = this.resourcePath;
+	var issuerContext = this._module && this._module.issuer && this._module.issuer.context || context;
```

Comment:
@michael-ciniawsky My purpose of adding issuer's context to the `publicPath` function is to give the   user a way to the resolved path trickily (eg. I can give a new relative path according to the issuer's context). So I prefer the issuer's context personally.

But I didn't fix the "useRelativePath" bug in this patch.

I notice that all the test cases are mocked cases, and there's no (good) way to find out problems caused by the webpack APIs (eg. `this._module`).
I'm not familiar with the webpack APIs. Maybe I need some help or advice to test it in a better way.

BTW, `this._module` has been deprecated by [officual docs](https://doc.webpack-china.org/api/loaders/#this-_module). Maybe it should be changed in some other way?